app/src/format.py

```diff
@@ -176,3 +177,15 @@ def _replace_bem_with_link(text: str) -> str:
         r'<a href="https://dhhs.michigan.gov/OLMWeb/ex/BP/Public/BEM/\2.pdf">\1</a>',
         text,
     )
+
+
+def _add_ellipses(chunk: Chunk) -> Chunk:
+    if chunk.num_splits != 0:
+        if chunk.split_index == 0:
+            chunk.content = f"{chunk.content}..."
+        elif chunk.split_index == chunk.num_splits:
+            chunk.content = f"...{chunk.content}"
+        else:
+            chunk.content = f"...{chunk.content}..."
+        chunk.content = re.sub(r"\.{4,}", "...", chunk.content)
+    return chunk
```

Comment:
Don't modify `chunk`, also you can reduce indentation and easier to interpret by `return`ing more:
```suggestion
    if chunk.num_splits == 0:
        return chunk

    if chunk.split_index == 0:
        return f"{chunk.content}..."
    
    if chunk.split_index == chunk.num_splits:
        return f"...{chunk.content}"

    return f"...{chunk.content}..."
```

I don't know why the following is needed -- I interpret this line as replace "...." with "...". Is there a test for this?
```   
    chunk.content = re.sub(r"\.{4,}", "...", chunk.content)
```

Reply:
Yes, the first test case has a period at the end of the line. In cases where there are more than 4 .'s, it'll replace it with the ellipses.

Reply:
I see. How about joining with a space in between: `" ".join(content, "...")`? I like this better for the end of sentences, such as `Some sentence ending with a period. ...` over `Some sentence ending with a period...` since it's obvious the sentence ended.